web/src/components/tasks/business-formation/BusinessFormation.test.tsx

```diff
@@ -1191,4 +1192,70 @@ describe("<BusinessFormation />", () => {
     expect(formationFormData.nonprofitTrusteesMethodSpecified).toEqual(undefined);
     expect(formationFormData.nonprofitAssetDistributionSpecified).toEqual(undefined);
   }, 60000);
+
+  describe("businessStartDate", () => {
+    afterEach(() => {
+      jest.useRealTimers();
+    });
+
+    it("when user is in US Eastern Timezone, initial value is current date in NJ", async () => {
+      const dateFormat = "MM/DD/YYYY";
+      const expectedDateString = getCurrentDateInNewJerseyFormatted(dateFormat);
+
+      const page = preparePage({
+        business: {
+          profileData: generateProfileData({
+            legalStructureId: "limited-liability-company",
+            businessPersona: "STARTING",
+          }),
+          formationData: generateEmptyFormationData(),
+        },
+        displayContent,
+      });
+      await page.fillAndSubmitBusinessNameStep("Pizza Joint");
+      expect(screen.getByTestId("date-businessStartDate")).toHaveValue(expectedDateString);
+    });
+
+    it("when user is in Phillipine Timezone, initial value is current date in NJ", async () => {
+      const mockDate = new Date("2020-04-13T00:00:00.000+08:00");
+      const expectedDateString = "04/12/2020";
+
+      jest.useFakeTimers();
+      jest.setSystemTime(mockDate);
+
+      const page = preparePage({
+        business: {
+          profileData: generateProfileData({
+            legalStructureId: "limited-liability-company",
+            businessPersona: "STARTING",
+          }),
+          formationData: generateEmptyFormationData(),
+        },
+        displayContent,
+      });
+      await page.fillAndSubmitBusinessNameStep("Pizza Joint");
+      expect(screen.getByTestId("date-businessStartDate")).toHaveValue(expectedDateString);
+    });
+
+    it("when user is in US Pacific Timezone, initial value is current date in NJ", async () => {
```

Comment:
[dust] I'd rather see this be in the Hawaii–Aleutian Time Zone, especially since you're not testing DST. Also US/Pacific isn't -08:00 all year round, most especially not on 4/13/2020 😉

Reply:
I think the test title should be something along the lines of... when the system date is 1 day before the date in NJ, the NJ date rendered. The current title makes me think we have some special logic around timezones